### DISCUSSION TITLE
Keep older versions of Meilisearch available via Brew

### DISCUSSION THREAD

**Discussion by schappim** [meilisearch/product#442] [Feedback & Feature Proposal]
TL;DR: Would be great to not have to install older versions from source (on macOS).

It would be great to enable older versions of Meilisearch to continue to be available via Brew until all language libraries have been updated to use the latest version (for example [meilisearch-rails](https://github.com/meilisearch/meilisearch-rails)).

Example of why this is an issue: 

I find myself in the position today where I accidently updated meilisearch via brew, and it installed meilisearch 0.26.1.

When I try to run my rails app that relies on 0.25.2, I get the error: 

```
meilisearch   | Error: Expected Meilisearch engine version: 0.25.2, current engine version: 0.26.1. To update Meilisearch use a dump.
```

I thought, not a problem, I'll just update the gem (unfortunately this is a no-go because the latest version of the gem still uses 0.25.2). 

Ok then, I'll install the older version of meilisearch via brew, however when I do a ```brew info meilisearch``` I get back: 

```
(base) admin@max meilisearch-0.25.2 % brew info meilisearch
meilisearch: stable 0.26.1 (bottled)
Ultra relevant, instant and typo-tolerant full-text search API
https://docs.meilisearch.com/
Not installed
From: https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/meilisearch.rb
License: MIT
==> Dependencies
Build: rust ✔
==> Caveats
To restart meilisearch after an upgrade:
  brew services restart meilisearch
Or, if you don't want/need a background service you can just run:
  /opt/homebrew/opt/meilisearch/bin/meilisearch --db-path /opt/homebrew/var/meilisearch/data.ms
==> Analytics
install: 690 (30 days), 2,168 (90 days), 5,983 (365 days)
install-on-request: 688 (30 days), 2,166 (90 days), 5,977 (365 days)
build-error: 0 (30 days)
```
So, now I have to install 0.25.2 via source. :-)

**Comment by gmourier** [comment]
Hi @schappim! Thanks for this feedback, indeed this is not convenient in that case.

@curquiza Do you have any idea of the feasibility and what it would mean for us to do this?

I guess another solution would be to not release the new version on brew until all the SDKs are up to date but that seems much less viable in the long run in my opinion.

Thanks!

**Reply by curquiza** [reply]
I'm 99% this is doable since other packages like postgres do it. But the problem is time, human resources, and priority of this regarding the other additions (as usual haha)
But yes, I think it's doable

**Comment by schappim** [comment]
Thanks @gmourier!

@curquiza you'll probably want this doco: https://docs.brew.sh/Versions#acceptable-versioned-formulae

**Reply by curquiza** [reply]
Thanks @schappim 😊
